### ISSUE TITLE
Min and Max modifiers don't work together

### ISSUE THREAD

**Issue by GreenImp** [GreenImp/rpg-dice-roller#165]
**Describe the bug**
When using both Min and Max modifiers together, only the first one works.
I'm not sure why, as they've got different names, so shouldn't be overriding each other. The other oddity is that if they are, the order they're written in shouldn't make a difference as they should always be run in the same order.

**To Reproduce**
The `min3` should set the `2` to a value of `3`, but it's not:
```
4d8min3max6: [6v, 2, 3, 6] = 17 
```

The `max6` should set the `7` to `6`, but it's not:
```
4d8max6min3: [5, 3^, 3, 7] = 18 
```

**Expected behavior**
I would expect both modifiers to work together. the above rolls should look like:

```
4d8min3max6: [6v, 3^, 3, 6] = 18

4d8max6min3: [5, 3^, 3, 6v] = 17
```

**Comment by GreenImp** [comment]
Oddly, this only seems to be affecting the documentation.
I've added tests to the project to make sure it works, and they're passing, and I've tested it in a separate project that includes the dice roller, and it's fine.

A bit more investigation required.

**Comment by GreenImp** [comment]
I've pushed up some changes in v4.3.1 that hopefully fixes this. I'll need to update the version used in the Vue component and test.

**Comment by GreenImp** [comment]
This has been fixed in 4.3.1, and the docs are now working again, but you may have to do a good hard cache clear on it (Vuepress or Github pages seems to have a strong cache setting).
